habitat/omnitruck-web/plan.sh

```diff
@@ -0,0 +1,17 @@
+pkg_name=omnitruck-web
+pkg_origin=chef-es
+pkg_version="0.1.0"
+pkg_maintainer="Chef Engineering Services <eng-services@chef.io>"
+pkg_license=('Apache-2.0')
+
```

Comment:
Consider adding a `pkg_binds` or `pkg_binds_optional` here as well so we can bind to the rack tcp socket automagically.

Eg:

```bash
pkg_binds=(
  [omnitruck]="port"
)
```
The value should be `port`, but it might be `app.port`. https://github.com/habitat-sh/core-plans/blob/master/scaffolding-ruby/lib/scaffolding.sh#L545-L547

The bind syntax is a little confusing, so I the example above may or may not do what I intend -- but the key thing is when you `load` or `start` this service you need to assign a matching name of `omnitruck`. https://www.habitat.sh/docs/run-packages-binding/

Also if you are successful with using a TCP socket, add a `pkg_exports` to this as well so that we can do the same magic from the nginx/proxy plan.